src/btree/bt_split.c

```diff
@@ -1395,7 +1395,8 @@ __split_multi_inmem(WT_SESSION_IMPL *session, WT_PAGE *orig, WT_MULTI *multi, WT
      * our caller will not discard the disk image when discarding the original page, and our caller
      * will discard the allocated page on error, when discarding the allocated WT_REF.
      */
-    WT_RET(__wt_page_inmem(session, ref, multi->disk_image, WT_PAGE_DISK_ALLOC, &page, &prepare));
+    WT_RET(__wt_page_inmem(
+      session, ref, multi->disk_image, multi->size, WT_PAGE_DISK_ALLOC, &page, &prepare));
```

Comment:
I think `multi->size` is the right variable to use here. However, a confirmation would be nice.

Reply:
I could dig into this and form an opinion, but it might be easier to pull in someone who is already familiar with the split code here.

Reply:
After discussing with @sulabhM, we found that the size allocated for creating a disk image for the multi structure is stored in multi->disk_image->mem_size (in __wt_rec_split). Therefore, I will utilize the same for cache tracking. 

Essentially, we do not plan to alter the cache tracking behavior when pages are created from a multi (disk image).